app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -464,24 +511,38 @@ public static void clickToolbarReload() {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.TOOLBAR, Value.RELOAD).queue();
     }
 
-    public static void clickToolbarShare() {
+    public static void clickToolbarShare(Context context) {
         TelemetryEvent.create(Category.ACTION, Method.SHARE, Object.TOOLBAR, Value.LINK).queue();
+
+        FirebaseEvent.create(Category.ACTION, Method.SHARE, Object.TOOLBAR, Value.LINK).queue(context);
     }
 
-    public static void clickAddToHome() {
+    public static void clickAddToHome(Context context) {
         TelemetryEvent.create(Category.ACTION, Method.PIN_SHORTCUT, Object.TOOLBAR, Value.LINK).queue();
+
+        FirebaseEvent.create(Category.ACTION, Method.PIN_SHORTCUT, Object.TOOLBAR, Value.LINK).queue(context);
     }
 
-    public static void clickToolbarCapture() {
+    public static void clickToolbarCapture(Context context) {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.TOOLBAR, Value.CAPTURE)
-                .extra(Extra.VERSION, Integer.toString(2))
+                .extra(Extra.VERSION, Integer.toString(TELEMETRY_VERSION))
```

Comment:
I'm not sure about this. This is about toolbarCapture V2, but not telemetry V2. (All telemetry has its own version, in this case, the position of toolbarcapture have changed in 2.0)

Reply:
Maybe rename TELEMETRY_VERSION to TOOL_BAR_CAPTURE_TELEMETRY_VERSION